src/pages/Jobs.js

```diff
@@ -19,8 +19,9 @@ export default function Jobs() {
   let [keyword, setKeyword] = useState(query.get(QUERYSTR_PREFIX));
 
   const getData = async () => {
-    //let url ="http://localhost:3001/jobs";
-    let url = `https://my-json-server.typicode.com/legobitna/Itviec/jobs`;
+    let url =`http://localhost:${process.env.REACT_APP_JSON_DEV_PORT}/jobs`;
```

Comment:
let url = `${process.env.BACKEND_URL}/jobs`

Reply:
solved it

Reply:
solved it